### ISSUE TITLE
Broken logout (not work)

### ISSUE THREAD

**Issue by emanuelb** [paragonie/airship#166]
in:
https://airship.paragonie.com/
clicking on 'logout' return 404 error, and not log-out the user.

**Comment by paragonie-scott** [comment]
Yeah, we need to rehaul that website. I just haven't had the chance to do so.
